src/Utility/ContentFormUtilities.php

```diff
@@ -51,10 +51,6 @@ public static function toggleFieldsWithCheckbox(array &$form, string $checkbox_f
    *   The value that makes the dependent field visible.
    */
   public static function toggleFieldByValue(array &$form, string $toggle_field, string $field, string $value): void {
-    if (!isset($form[$toggle_field])) {
```

Comment:
It was removed to be used in hook_inline_entity_form_entity_form_alter() where elements from the parent form don't exist in the `$form` variable.